ai_oca_bridge/models/ai_bridge.py

```diff
@@ -32,20 +35,40 @@ class AiBridge(models.Model):
     name = fields.Char(required=True, translate=True)
     active = fields.Boolean(default=True)
     description = fields.Html(translate=True)
-    model_id = fields.Many2one(
-        "ir.model",
-        string="Model",
-        domain=[("transient", "=", False)],
+    user_id = fields.Many2one(
+        "res.users",
+        default=lambda self: self.env.user,
+        help="The user that will be shown when executing this AI bridge.",
+    )
+    payload_type = fields.Selection(
+        [
+            ("record", "Record"),
+            ("record_v0", "Record v0"),  # Deprecated, use 'record' instead
+        ],
         required=True,
-        ondelete="cascade",
-        help="The model to which this bridge is associated.",
+        default="record",
     )
-    model = fields.Char(
-        related="model_id.model",
-        string="Model Name",
+    result_type = fields.Selection(
+        [
+            ("none", "None"),
+            ("message", "Post a Message"),
+            ("action", "Action"),
+        ],
+        required=True,
+        default="none",
+        help="Defines the type of result expected from the AI system.",
     )
-    domain = fields.Char(
-        string="Filter", compute="_compute_domain", readonly=False, store=True
+    result_kind = fields.Selection(
+        [("immediate", "Immediate"), ("async", "Asynchronous")],
+        default="immediate",
+        help="Defines how the result from the AI system is processed. "
```

Comment:
Could the following be indicated?
* Sync: For processes that require you to wait for the connection to end. 
* Async: For processes that do not require the process to end to continue. The user will not receive a pop-up notification, but the execution will be logged.